ext/tests/expand_params_internal.phpt

```diff
@@ -5,10 +5,10 @@ Removing the `post` callback avoids the segfault. The segfault is actually durin
 from `zend_observer_fcall_end_all` (https://github.com/php/php-src/blob/php-8.2.8/Zend/zend_observer.c#L291).
 When it traverses back through zend_execute_data, the top-level frame appears corrupted, and any attempt to reference
 it is what causes the segfault.
+--SKIPIF--
+<?php if (PHP_VERSION_ID < 80200) die('skip requires PHP >= 8.2'); ?>
```

Comment:
This test should work with 8.1 (which is where observing internal functions was introduced). Can you remove the `null` type-hint instead to make this pass?

Reply:
In PHP 8.0-8.1 will get:
`Fatal error: Null can not be used as a standalone type in D:\a\opentelemetry-php-instrumentation\opentelemetry-php-instrumentation\ext\tests\expand_params_internal.php on line 5`

Because is XFAIL-ed will pass the test. For this i removed XFFAIL and added to skip if PHP lower than 8.2

Reply:
sorry that was my mistake. Observing internal functions was introduced in 8.2.